### PR TITLE
Initial Stub Connector

### PR DESCRIPTION
**Pull request by aakashnshah**

### CODE REVIEW

File: QUICKSTART.md [hunk 1]
```diff
@@ -131,11 +114,10 @@ curl -X POST -d @kafka-connect-aws-cloudwatch.json http://localhost:8083/connect
 
 ### Trace Logging
 
-Both the sink and source connectors have trace logs which show in greater detail what records are passing through them. To enable them, add the following lines to your log4j.properties and restart the connect worker (*important*: we have to restart the worker otherwise these changes will not be picked up):
+The source connector has trace logs which show in greater detail what records are passing through them. To enable them, add the following lines to your log4j.properties and restart the connect worker (*important*: we have to restart the worker otherwise these changes will not be picked up):
```
Comment (gharris1727) [inline QUICKSTART.md]: passing through ~them~ _it_

File: src/test/java/io/confluent/connect/aws/cloudwatch/AwsCloudWatchSourceConnectorTest.java [hunk 2]
```diff
@@ -85,10 +85,7 @@ public void version() {
     assertTrue(connector.version().matches("^(\\d+\\.)?(\\d+\\.)?(\\*|\\d+)(-\\w+)?$"));
   }
 
-  /**
-   * Start the connector, via the {@link AwsCloudWatchSinkConnector#doStart()} method that uses our mock
```
Comment (gharris1727) [inline src/test/java/io/confluent/connect/aws/cloudwatch/AwsCloudWatchSourceConnectorTest.java]: This looks like another typo in the archetype. You can leave this deleted.

File: README.md [hunk 3]
```diff
@@ -1 +1,34 @@
-# kafka-connect-aws-cloudwatch
\ No newline at end of file
+# Introduction
+
+This project provides connectors for Kafka Connect to read and write data to AWSCloudWatch.
```
Comment (levzem) [inline README.md]:
```suggestion
This project provides connectors for Kafka Connect to read data from AWSCloudWatch.
```

File: config/aws-cloudwatch-source-quickstart.properties [hunk 4]
```diff
@@ -0,0 +1,3 @@
+name=aws-cloudwatchSourceConnector
```
Comment (levzem) [inline config/aws-cloudwatch-source-quickstart.properties]: this looks funny, either use - or camelcase

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+                <groupId>io.confluent</groupId>
+                <artifactId>kafka-connect-maven-plugin</artifactId>
+                <version>${confluent.hub.packaging.version}</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>kafka-connect</goal>
+                        </goals>
+                        <configuration>
+                            <title>Kafka Connect AWSCloudWatch Connector</title>
```
Comment (levzem) [inline pom.xml]:
```suggestion
                            <title>Kafka Connect AWSCloudWatch Source Connector</title>
```

**Pull request merged**